tools/system_libs.py

```diff
@@ -406,7 +406,8 @@ def build_with_ninja(self, build_dir, libname):
     ensure_sysroot()
     utils.safe_ensure_dirs(build_dir)
 
-    cflags = self.get_cflags()
+    source_dir = utils.path_from_root()
+    cflags = self.get_cflags() + [f'-fdebug-prefix-map={source_dir}=/buildbot']
```

Comment:
How about `/emscripten` or `/emsdk/emscripten/`?    Good to have emscirpten in the directory name I think this since the path we are replacing here is the emscripten root, right?

What about windows?  Should we use backslashes there?

Reply:
Or "/builtbot/emscripten"

Reply:
Yeah I think /emsdk/emscripten is good. If there are other emsdk-install-relative paths they can be consistent.
WRT Windows, I think could go either way. forward slashes are perfectly fine in Windows, and I assume the substitution rules in the debugger don't care one way or the other. If we go to a model where we build the system libs on one platform and use the same ones on all platforms, then the rest of the path will certainly have backslashes anyway. Otherwise the slashes will match the command line we use to build them, which we also have control over here.